### DISCUSSION TITLE
Just CANNOT GET IT TO WORK!

### DISCUSSION THREAD

**Discussion by pekflyer** [community/community#56290] [GitHub Education]
### Hi everyone,

I have spent an hour trying to get myself validated. 

I have provided my certificate of enrolment, both sides of the student card with validity on it.

The system thinks I am a fraud and there is no direct communication channel I can get this simple step ratified. 

IS there a way to get this to work?  It is very frustrating.

**Comment by samus-aran** [comment]
Hi @pekflyer 

Often this common pre-processing warning is displayed because the application detects differences between your profile information and the evidence you’ve supplied. We do request that students can demonstrate that their GitHub account corresponds with the evidence given. The verification process looks at your GitHub username, email, and name (as well as other points) to cross-reference the evidence submitted. Do you use a variation of your name or a nickname?  It’s these small but important points to consider when applying that can make a much smoother process.

Ensure your image (photo or upload) is legible. If you are unable to read its text on the image itself then the verification process may not be able to either. If you are taking a photo of a screen try to have it as large a resolution as possible - take a photo of the evidence on a monitor rather than a phone perhaps. Does your image have reflective surface, perhaps include written copy alongside the evidence if the text & logos overlap.

Hope this helps. 👍
